### ISSUE TITLE
Create styled 404 page

### ISSUE THREAD

**Issue by Cleop** [dwyl/dwyl-site#313]
Create styled 404 page as per @harrygfox 's designs:
![image](https://user-images.githubusercontent.com/16775804/30208937-8b1ab27e-948d-11e7-8923-e7997a9907b4.png)

**Comment by ZooeyMiller** [comment]
I'm going to look into this now. I'm not 100% on setting up a custom 404 on GH pages, hence the 2h time estimate so I can work that out.

**Comment by ZooeyMiller** [comment]
@harrygfox do you have the asset of the broken dwyl heart that I can use?

**Comment by markwilliamfirth** [comment]
![broken heart](https://user-images.githubusercontent.com/5732209/31081976-728a197e-a785-11e7-8a12-3c1b254508d2.png)

@ZooeyMiller @finnhodgkin have added you to the invision design project via your dwyl email

**Comment by SimonLab** [comment]
Just wondering, is there a way for the users to get redirected to the main page? Is the logo a clickable link to the home page? Do we want a link? @markwilliamfirth @Cleop @ZooeyMiller ?

**Comment by ZooeyMiller** [comment]
@SimonLab on the original design the 404 is just the body of the main outline of the page. 

Here's a grab of the original invision design.

![dwyl preview mode - invision - google chrome_043](https://user-images.githubusercontent.com/21139983/31083006-5485c75e-a788-11e7-82e9-e86f09a7c357.png)

**Comment by markwilliamfirth** [comment]
This works!

**Comment by harrygfox** [comment]
![image](https://user-images.githubusercontent.com/7805691/31552506-a273422a-b02f-11e7-8100-30021ca89c76.png)
pixel-y image has a different colour to the background.

**Comment by iteles** [comment]
@harrygfox We'd love to get a `png` of the heart we could use for this! ❤️

**Comment by harrygfox** [comment]
@iteles indeed! it's [here in both `.png` and `.svg`](https://projects.invisionapp.com/d/main#/projects/prototypes/11013483/assets)

**Comment by iteles** [comment]
You're an actual star 🌟 I was still writing the email to you to request these!

**Comment by rub1e** [comment]
@sophielevens another one for you, if you can manage it?
